deploy/osd-managed-resources/get-managed-resources.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Can we introduce a few options to the script? 
* --output, -o <yaml or configmap>
* --all, -a <output ALL resources, defaults to just namespaces>
* --path, -p <path to artifact>

Reply:
Thanks, I added these options to the python script.